### ISSUE TITLE
READ_PHONE_STATE permission

### ISSUE THREAD

**Issue by AlexeyFreelancer** [felipecsl/QuickReturn#27]
Hi. Your library added permission READ_PHONE_STATE and app has a problem during install (e.g. avast antivirus detect it how dangerous app).

Please take a look my log manifest-merger-release-report.txt
```
android:uses-permission#android.permission.WRITE_EXTERNAL_STORAGE
IMPLIED from AndroidManifest.xml:2:1 reason: com.felipecsl.quickreturn has a targetSdkVersion < 4
android:uses-permission#android.permission.READ_PHONE_STATE
IMPLIED from AndroidManifest.xml:2:1 reason: com.felipecsl.quickreturn has a targetSdkVersion < 4
android:uses-permission#android.permission.READ_EXTERNAL_STORAGE
IMPLIED from AndroidManifest.xml:2:1 reason: com.felipecsl.quickreturn requested WRITE_EXTERNAL_STORAGE
```

**Comment by felipecsl** [comment]
Thank you for the heads up, will remove that

**Comment by felipecsl** [comment]
I don't see this anywhere in the repo. Are you sure about it?

**Comment by AlexeyFreelancer** [comment]
Please take a look at answer https://code.google.com/p/android/issues/detail?id=81287#c11

> the targetSdkVersion is defaulted to the minSdkVersion (3 in this case) which prompted the manifest merger to add those permissions.

Could you add uses-sdk to the AndroidManifest.xml:
```<uses-sdk android:minSdkVersion="10" android:targetSdkVersion="21"/>```

**Comment by felipecsl** [comment]
Released to maven central with version 1.5.1
It should be available in a few minutes

**Comment by AlexeyFreelancer** [comment]
Works perfect now, many thanks.
![perm](https://cloud.githubusercontent.com/assets/1579687/5813483/d4708618-a086-11e4-80db-0609b1decd39.png)
